src/lib/fragments/AgGrid.react.js

```diff
@@ -1120,8 +1120,12 @@ export default class DashAgGrid extends Component {
             }));
             return (params) => {
                 for (const {test, style} of tests) {
-                    if (test(params)) {
-                        return style;
+                    if (params) {
+                        if (params.data) {
+                            if (test(params)) {
+                                return style;
+                            }
+                        }
                     }
```

Comment:
It's syntax sugar, but you can remove a nested `if` with:
```suggestion
                   if (params?.data) 
                        if (test(params)) {
                            return style;
                        }
                    }
                    
```

Reply:
This doesn't lint, otherwise I would have done it.

Reply:
Plausibly because we're supporting old versions of JS. Feel free to ignore this comment then and merge!